python/jupytergis_lab/jupytergis_lab/notebook/tests/test_api.py

```diff
@@ -46,12 +46,13 @@ def test_remove_nonexistent_layer_raises(self):
             self.doc.remove_layer("foo")
 
 
-def test_save_as(tmp_path):
+def test_untitled_doc(tmp_path):
     os.chdir(tmp_path)
 
-    doc = GISDocument()
-    assert not list(tmp_path.iterdir())
+    GISDocument()
+    assert len(list(tmp_path.iterdir())) == 1
+    assert (tmp_path / "untitled.jGIS").is_file()
 
-    fn = "test.jgis"
-    doc.save_as(fn)
-    assert (tmp_path / fn).is_file()
+    GISDocument()
+    assert len(list(tmp_path.iterdir())) == 2
+    assert (tmp_path / "untitled1.jGIS").is_file()
```

Comment:
This seemed like the natural way to test this, but I believe because there is no JupyterLab instance running, the JavaScript code which actually creates the files is not being exercised. Is there a better way to test this?